src/internal/SelectionLabel.js

```diff
@@ -0,0 +1,38 @@
+// @flow weak
+
+import React, { PropTypes } from 'react';
+import { createStyleSheet } from 'jss-theme-reactor';
+import classNames from 'classnames';
+
+export const styleSheet = createStyleSheet('SelectionLabel', () => {
+  return {
+    root: {
+      marginLeft: -12,
+      marginRight: 16, // used for row presentation of radio/checkbox
+      display: 'flex',
+      alignItems: 'center',
+      cursor: 'pointer',
+    },
+  };
+});
+
+export default function SelectionLabel(props, context) {
+  const { label, labelClassName, children } = props;
+  const classes = context.styleManager.render(styleSheet);
+  return (
+    <label className={classNames(classes.root, labelClassName)} role="presentation">
```

Comment:
We could make further refactorisation. That block is duplicated between `Checkbox` and `Radio`.
```jsx
  if (label) {
    switchProps['aria-label'] = label;
    return (
      <SelectionLabel label={label} className={labelClassName}>
        <SwitchBase {...switchProps} />
      </SelectionLabel>
    );
  }

  return <SwitchBase {...switchProps} />;
```